.github/workflows/ci.yml

```
@@ -29,7 +29,7 @@ jobs:
           node-version: 22
           cache: pnpm
       - run: pnpm install --frozen-lockfile
-      - run: pnpx pkg-pr-new publish --comment=off ./packages/kit
+      - run: pnpx pkg-pr-new publish --comment=off ./packages/kit ./packages/adapter-* ./packages/enhanced-img ./packages/package
```

could we just publish everything?
```suggestion
      - run: pnpx pkg-pr-new publish --comment=off ./packages/*
```